g3/generate/pr/prompts/creator.py

```diff
@@ -1,8 +1,59 @@
 from typing import Optional
 
 from g3.domain.message_tone import MessageTone
+from g3.generate.pr.prompts.template import pr_template
+from g3.git.gitinfo import GitInfo
+from g3.main import config
 
 
 class Creator:
-    def create(self, tone: MessageTone, jira: Optional[str] = None, include: Optional[str] = None) -> list:
-        raise NotImplementedError
+    def __init__(self):
+        self.git_info = GitInfo()
+
+    def create(self, tone: MessageTone, include: Optional[str] = None) -> list:
+        system_messages = self.create_system_messages(tone, include)
+        return system_messages + self.user_messages
+
+    @property
+    def user_messages(self) -> list:
+        commits: list = []  # TODO: replace with actual commits for git info
```

Comment:
Let's move the definition of the list of commit messages in the initializer based on the `GitInfo()`. For example:
```
    def __init__(self):
        self.git_info = GitInfo()
        self.commit_messages: List[str] = self.git_info.commit_messages
```